examples/advanced/sklearn-kmeans/utils/prepare_job_config.py

```diff
@@ -195,7 +195,7 @@ def _update_client_config(config: dict, args, site_name: str, site_indices):
 
 
 def _update_server_config(config: dict, args):
-    config["workflows"][0]["args"]["min_clients"] = args.site_num
+    config["min_clients"] = args.site_num
```

Comment:
This doesn't seem to work

Reply:
I also changed the config_fed_client.json in this PR for easier automation.
I put the min_clients on top and use `{min_clients}` in component to do substitution